src/workers/parser/getSymbols.js

```diff
@@ -5,14 +5,15 @@
 // @flow
 
 import { traverseAst } from "./utils/ast";
-import { isVariable, isFunction } from "./utils/helpers";
+import { isVariable, isFunction, isJSXElement } from "./utils/helpers";
 import { inferClassName } from "./utils/inferClassName";
 import * as t from "babel-types";
 import getFunctionName from "./utils/getFunctionName";
 
 import type { Source } from "debugger-html";
 import type { NodePath, Node, Location as BabelLocation } from "babel-traverse";
 let symbolDeclarations = new Map();
+var hasJSX = false;
```

Comment:
this should be defined in the function it's used
also, we use `let` and `const` :)